investpy/stocks.py

```diff
@@ -1506,3 +1506,176 @@ def search_stocks(by, value):
     search_result.reset_index(drop=True, inplace=True)
 
     return search_result
+
+
+def get_stock_earnings(stock, country):
+    """
+        This function retrieves the earnings data from the specified stock. Earnings data include date of the
```

Comment:
Minor/Silly fix, but can you un-tab this so as to align it with all the docstrings? Thanks!